features/promotion/receiving_percentage_discount_on_products_from_specific_taxon.feature

```diff
@@ -0,0 +1,49 @@
+@promotion
+Feature: Receiving percentage discount on products from specific taxon
+    In order to pay less while buying goods from promoted taxons
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And store classifies its products as "T-Shirts" and "Mugs"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And it belongs to "Mugs"
+        And there is a promotion "T-Shirts promotion"
+        And it gives "20%" percentage discount to every order item from "T-Shirts" taxon
+
+    @todo
+    Scenario: Receiving percentage discount on a single item from promoted taxon
```

Comment:
``promoted taxon`` does not sound right.

Receiving percentage discount on a single item from specific taxon

To be honest I am not sure how to associate product with taxon:

* under taxon
* from taxon
* classified as taxon

...